prow/config.yaml

```diff
@@ -1846,29 +1846,60 @@ periodics:
   name: ci-tekton-pipeline-upgrade-tests
   labels:
     preset-presubmit-sh: "true"
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+    pipeline-kind-e2e: "true"
   agent: kubernetes
   extra_refs:
   - org: tektoncd
     repo: pipeline
     base_ref: main
     path_alias: github.com/tektoncd/pipeline
   spec:
+    affinity:
+      podAntiAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+                - key: pipeline-kind-e2e
+                  operator: Exists
+            topologyKey: kubernetes.io/hostname
+    nodeSelector:
+      cloud.google.com/gke-ephemeral-storage-local-ssd: "true"
+      cloud.google.com/gke-nodepool: n2-standard-4-kind
+    tolerations:
+      - key: kind-only
+        operator: Equal
+        value: "true"
+        effect: NoSchedule
     containers:
     - image: gcr.io/tekton-releases/dogfooding/test-runner@sha256:01bace96516d22d70c092ff9c01f9ffe7af806c7475157dcaa7cdb45d84bd68b # golang 1.18.7
       imagePullPolicy: Always
       command:
         - /usr/local/bin/entrypoint.sh
       args:
-      - "--scenario=kubernetes_execute_bazel"
-      - "--clean"
-      - "--job=$(JOB_NAME)"
-      - "--repo=github.com/tektoncd/pipeline=main"
-      - "--root=/go/src"
       - "--service-account=/etc/test-account/service-account.json"
-      - "--upload=gs://tekton-prow/pr-logs"
       - "--" # end bootstrap args, scenario args below
       - "--" # end kubernetes_execute_bazel flags (consider following flags as text)
+      - "/usr/local/bin/kind-e2e"
+      - "--k8s-version"
+      - "v1.23.x"
+      - "--nodes"
+      - "3"
+      - "--e2e-script"
+      - "./test/e2e-tests.sh"
+      - "--e2e-env"
+      - "./test/e2e-tests-kind-prow.env"      
```

Comment:
Does this run e2e tests in addition to upgrade tests? (I think upgrade tests include e2e tests)